proto/config/netconfig.proto

```diff
@@ -55,6 +55,10 @@ message NetworkAdapter {
   // valid vlan id range: 2 - 4093
   // vlan id 1 is implicitly used by linux bridges
   uint32 access_vlan_id = 41;
+
+  // allow AppInstance to discover other AppInstances
+  // attached to its network instances. default is false
```

Comment:
Shouldn't this say "to this network instance" in singular?
Or to the network instance attached to this adapter?

Reply:
renamed to "this network instance"